### PR TITLE
Added digital post test command

### PR DESCRIPTION
**Pull request by rimi-itk**
Adds a command for testing sending digital post.

### CODE REVIEW

File: modules/os2forms_digital_post/README.md [hunk 1]
```diff
@@ -58,11 +58,20 @@ graphical overview of jobs in the queue.
 
 ## Beskedfordeler
 
-Thie digital post module depends on [Beskedfordeler for
+The digital post module depends on [Beskedfordeler for
 Drupal](https://github.com/itk-dev/beskedfordeler-drupalon) to get get
```
Comment (jekuaitk) [inline modules/os2forms_digital_post/README.md:62]:
```suggestion
Drupal](https://github.com/itk-dev/beskedfordeler-drupal) to get get
```

**Review by jekuaitk — APPROVED** [review]
Looks good to me.

**Comment by rimi-itk** [comment]
> Minor improvements and a full test suite. That i will test.

@skifter, neither @jekuaitk or I understand what you're saying …

**Comment by skifter** [comment]
@rimi-itk Approved :-)

**Pull request merged**